tests/core/test_evaluate.py

```diff
@@ -49,3 +52,21 @@ def test_Evaluator() -> None:
     for expr, vals, expected in test_cases:
         for func in eval_funcs:
             assert func(expr, vals) == expected
+
+
+def test_Transformer() -> None:
+    """Test defining and using a Transformer."""
+    from protosym.core.tree import funcs_symbols
+    from protosym.core.evaluate import Evaluator, Transformer
```

Comment:
Why are these imported at the test level and not the module level?

Reply:
`Evaluator` is already imported above at the module level.

Reply:
Moved.